class.dynamic_cdn.php

```diff
@@ -43,14 +43,18 @@ public function init() {
 		$filter_priority = apply_filters( 'dynamic_cdn_filter_priority', 10 );
 		$action_priority = apply_filters( 'dynamic_cdn_action_priority', 10 );
 		
-		$this->uploads_only = apply_filters( 'dynamic_cdn_uploads_only', false );
+		if ( defined( 'DYNCDN_UPLOADS_ONLY' ) ){
```

Comment:
Let's hook this via the Tenup plugin or otherwise. I'd feel better if we weren't modifying a third party plugin, even though it's forked. The intent is to keep up with upstream.